src/Scanner.js

```diff
@@ -214,6 +223,21 @@ class DefaultScanner {
         return this.mode === 'stopped';
     }
 
+    /**
+     * Gets the forced form only for the first call, subsequent calls return null
+     * @returns {HTMLFormElement|HTMLElement|null}
```

Comment:
HTMLFormElement inherits from HTMLElement, so using both is not needed here. You should go stricter when possible (not here) but otherwise it's ok to just use the ancestor. I'm pushing the change as I wanted to `run tsc` to verify.

```suggestion
     * @returns {HTMLElement|null}
```